cmd/livepeer_cli/wizard_rounds.go

```diff
@@ -2,12 +2,20 @@ package main
 
 import (
 	"fmt"
+
+	"github.com/golang/glog"
 )
 
 func (w *wizard) currentRound() string {
 	return httpGet(fmt.Sprintf("http://%v:%v/currentRound", w.host, w.httpPort))
 }
 
 func (w *wizard) initializeRound() {
-	httpPost(fmt.Sprintf("http://%v:%v/initializeRound", w.host, w.httpPort))
+	initialized := httpGet(fmt.Sprintf("http://%v:%v/roundInitialized", w.host, w.httpPort))
```

Comment:
This bit seems separate from the ability to set gas price. Maybe split the commit?

Also, would it be more robust to check if the round is initialized within the `/initializeRound` http handler, rather than doing a separate check via POST? That way there's no possibility of a bug, CSRF, etc repeatedly calling `initializeRound` and draining funds.